src/components/navBar/NavBarView.js

```diff
@@ -0,0 +1,343 @@
+import React from 'react';
+import { HashRouter, NavLink } from 'react-router-dom';
+import {
+  AppBar,
+  Button,
+  Toolbar,
+  Tooltip,
+  withStyles,
+} from '@material-ui/core';
+import classnames from 'classnames';
+import DropdownMenu from './components/DropdownMenu';
+
+const drawerWidth = 240;
+// const FENCE_LOGIN_URL = process.env.FENCE_LOGIN_URL;
+// const FENCE_LOGIN_URL = process.env.REACT_APP_LOGIN_URL;
+const BACKEND_GETUSERINFO_API = 'https://k9dc.essential-dev.com/fence/login/';
+
+const NavBar = ({
+  classes, isSidebarOpened, navBarData, navBarCartData, navBarstyling,
+}) => {
+  // const theme = useTheme();
+  const [authState, setAuthState] = React.useState({
+    isAuthorized: localStorage.getItem('isAuthorized') === 'true',
+  });
+
+  // Similar to componentDidMount and componentDidUpdate:
+  // Empty second argument of react useEffect will avoid the infinte loop that
+  // caused due to component update
+  const [clickedEl, setClickedEl] = React.useState(null);
+
+  function handleButtonClickEvent(eventName) {
+    setClickedEl(eventName);
+  }
+
+  React.useEffect(() => {
+    const values = {};
```

Comment:
Can you clean up or comment it out this as we are no longer using it

Reply:
Done